docs/xbcrypt-options.md

```diff
@@ -1,56 +1,70 @@
 # The xbcrypt command-line options
 
-The `xbcrypt` binary has the following command line options:
-
-### encrypt-algo
+Usage: 
+ 
+ ```{.bash data-prompt=text}
```

Comment:
```suggestion
```{.bash data-prompt="$"}
```

Reply:
This code block is broken in the preview.

Reply:
changed to ```{.text .no-copy}
The code is pseudo-code and should not work.